src/Hocon/HoconRoot.cs

```diff
@@ -695,17 +916,21 @@ public virtual IList<string> GetStringList(string path)
         /// <returns>The list of string values defined in the specified path.</returns>
         public virtual IList<string> GetStringList(string path, IList<string> @default)
         {
-            return WrapWithValueException(path, () => GetStringList(HoconPath.Parse(path), @default));
+            if (TryGetNode(path, out var value))
+                if (value.TryGetStringList(out var result))
+                    return result;
+
+            return @default;
         }
 
         /// <inheritdoc cref="GetStringList(string)" />
-        public virtual IList<string> GetStringList(HoconPath path, IList<string> @default = null)
+        public virtual IList<string> GetStringList(HoconPath path, IList<string> @default)
         {
-            return WrapWithValueException(path.ToString(), () =>
```

Comment:
These designs, using the `Try` methods instead of `WrapWithValueException`, are a major improvement.